src/components/subpages/team/Team.astro

```diff
@@ -0,0 +1,113 @@
+---
+import { getLangFromUrl, useTranslations } from '../../../i18n/utils';
+import TeamMember from './TeamMember.astro';
+import PK from '../../../images/philippe_kobel.png';
+import BM from '../../../images/beatrice_mandanis.jpeg';
+import SP from '../../../images/stephanie_prior.jpg';
+import RP from '../../../images/robert_perez.jpg';
+import SG from '../../../images/stephane_goyette.jpeg';
+import TG from '../../../images/thierry_gerez.jpeg';
+import YD from '../../../images/yves_debernardi.jpeg';
+import KP from '../../../images/kim.webp';
+import HT from '../../../images/hagop.webp';
+
+const lang = getLangFromUrl(Astro.url);
+const t = useTranslations(lang);
+---
+
+<div
+  id="team"
+  class="container-fluid pb-5"
+  style={{ backgroundColor: '#c7f1bf' }}
+>
+  <div class="container">
+    <div class="flexbox">
+      <h1 class="text-center my-4">{t('Team')}</h1>
+      <div class="my-5">
+        <p>
+          {t('TEAM_DESCRIPTION')}
+        </p>
+        <p>
+          {t('TEAM_ACKNOLEDGEMENTS')}
+        </p>
+      </div>
+      <div class="container">
+        <div class="row">
+          <div class="col col-sm-4 col-lg-3 col-xl-2">
+            <TeamMember
+              img={PK}
+              name="Philippe Kobel"
+              roles={[
+                t('COORDINATOR'),
+                t('PHYSICS_TEACHER_MALE'),
+                'Gymnase du Bugnon',
+              ]}
+            />
+          </div>
+          <div class="col col-sm-4 col-lg-3 col-xl-2">
+            <TeamMember
+              img={KP}
+              name="Kim Phan"
+              roles={[t('DEVELOPER_FEMALE'), 'Graasp Association']}
+            />
+          </div>
+          <div class="col col-sm-4 col-lg-3 col-xl-2">
+            <TeamMember
+              img={HT}
+              name="Hagop Taminian"
+              roles={[t('DEVELOPER_MALE'), 'Graasp Association']}
+            />
+          </div>
+          <div class="col col-sm-4 col-lg-3 col-xl-2">
+            <TeamMember
+              img={SP}
+              name="Stéphanie Prior"
+              roles={[t('CHEMISTRY_TEACHER_FEMALE'), 'Gymnase du Bugnon']}
+            />
+          </div>
+          <div class="col col-sm-4 col-lg-3 col-xl-2">
+            <TeamMember
+              img={RP}
+              name="Robert Perez"
+              roles={[t('CHEMISTRY_TEACHER_MALE'), 'Gymnase du Bugnon']}
+            />
+          </div>
+          <div class="col col-sm-4 col-lg-3 col-xl-2">
+            <TeamMember
+              img={TG}
+              name="Thierry Gerez"
+              roles={[t('CHEMISTRY_TEACHER_MALE')]}
+            />
+          </div>
+          <div class="col col-sm-4 col-lg-3 col-xl-2">
+            <TeamMember
+              img={BM}
+              name="Béatrice Mandanis"
+              roles={[t('CHEMISTRY_TEACHER_FEMALE'), 'stepha']}
```

Comment:
I believe 'stepha' here is an error

Reply:
it is correct! it's the real name of the school, at least what is written in Philippe's school.